README.md

```diff
@@ -618,15 +619,12 @@ Expected return for type `mapping_votes` below
 
 ```json
 {
-    "Terminology": "tm--2VjOxekLP8m28EPRqk95",
-    "code": "TEST_0001",
+    "Terminology": "tm-C8IP8Cw_0M_hHWeLl5WP3",
+    "code": "female_ex",
+    "mapped_code": "female",
```

Comment:
I much prefer the more meaningful example codes. It really makes it much easier to discuss when we are reviewing them.